src/map_object.c

```diff
@@ -1819,14 +1821,14 @@ ObjectEvent* sub_0205FA98(u16 id, int num_templates, ObjectEvent* templates) {
     return NULL;
 }
 
-BOOL ObjectEventTemplate_ScriptIdIsFFFF(ObjectEvent* template) {
-    u16 script = (u16)(u32)ObjectEventTemplate_GetScript(template);
+BOOL ObjectEvent_ScriptIDIsUnset(ObjectEvent* template) {
+    u16 script = (u16)(u32)ObjectEvent_GetScript(template);
```

Comment:
:skull:

Reply:
yes this is required to match, GF moment...